ultralytics/engine/validator.py

```
@@ -186,6 +186,10 @@ def __call__(self, trainer=None, model=None):
             self.dataloader = self.dataloader or self.get_dataloader(self.data.get(self.args.split), self.args.batch)
 
             model.eval()
+            if self.data["channels"] != 1:
+                LOGGER.warning(f"Model channels mismatch: {self.data['channels']} != 1")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)